<launch>

  <node pkg="lms1xx" type="LMS1xx_node" name="LMS1xx_node"/>
  <node pkg="abd" type="abd_test" name="abd_test"/>
  <node pkg="scan_breakpoint" type="scan_breakpoint" name="scan_breakpoint"/>
  <node pkg="obstacle_detector" type="scans_merger_node" name="scans_merger">
  </node>
  <node pkg="obstacle_detector" type="obstacle_extractor_node" name="obstacle_extractor"/>
  .</launch>
